sirepo/pkcli/job_supervisor.py

```diff
@@ -48,10 +48,10 @@ def default_command():
         static_url_prefix=sirepo.job.LIB_FILE_URI + '/',
     )
     server = tornado.httpserver.HTTPServer(app)
-    server.listen(cfg.port, cfg.ip)
+    server.listen(cfg.port, cfg.host)
```

Comment:
this could be host or ip, but I prefer ip, since we know 127.0.0.1 works on all hosts. We don't know what host works on an arbitrary (curl) installation.